### DISCUSSION TITLE
Issues with eBPF on Liquorix kernels

### DISCUSSION THREAD

**Discussion by markozajc** [evilsocket/opensnitch#972] [Q&A] [answered]
I'm using Opensnitch 1.6.0 on Debian trixie/testing with Liquorix kernel 6.3.8-1-liquorix-amd64. Liquorix seems to disable some of the tracing capabilities of Linux, which means that `/sys/kernel/debug/tracing/events/syscalls/` doesn't exist. As a result, I get this when starting Opensnitch in eBPF mode:
```
[eBPF events] error enabling tracepoint tracepoint/syscalls/sys_enter_execve: cannot read tracepoint id "/sys/kernel/debug/tracing/events/syscalls/sys_enter_execve/id": open /sys/kernel/debug/tracing/events/syscalls/sys_enter_execve/id: no such file or directory 
[eBPF events] error enabling tracepoint tracepoint/syscalls/sys_enter_execveat: cannot read tracepoint id "/sys/kernel/debug/tracing/events/syscalls/sys_enter_execveat/id": open /sys/kernel/debug/tracing/events/syscalls/sys_enter_execveat/id: no such file or directory 
```
On the other hand, the firewall seems to work just fine, and programs making network requests are detected normally.
How big of a problem is this? If this allows programs to bypass the firewall I can just switch to Debian's kernel builds which have `/sys/kernel/debug/tracing/events/syscalls/`, but if it's not a problem I might as well continue using Liquorix.

**Comment by akronym0** [comment, marked as answer]
See also #774

**Reply by markozajc** [reply]
Thanks! Seems like my thread is indeed a duplicate.
